azure-pipelines.yml

```
@@ -0,0 +1,21 @@
+variables:
+  group: docker-creds
```

```suggestion
  - group: docker-creds
```

Pretty sure this only works when it's a list item.

---

I have CI [enabled on my fork](https://dev.azure.com/ecstaticmorse2/crates-build-env/_build?definitionId=2), so this does indeed work.

---

Hmm, you don't actually test uploading though. Can you check if that line creates a `group` environment variable with `docker-creds` content or actually imports those variables from the variable group?

---

Looking at it again and it does seem wrong. CI build on master is going now. I have tested the publish job (I deleted all builds before the pull request was opened) and it did succeed. Perhaps it's because I manually linked the variable group in the web UI?

---

...and it failed. Good catch! Not sure how I missed this :/.